getting-started.md

```diff
@@ -110,6 +110,7 @@ development environment.
 Set the type of board you want to use.
 ```
 export PLATFORM=opsis
+export TARGET=hdmi2usb
```

Comment:
You only want to set `TARGET=hdmi2usb` if you are actually working on the JPEG encoder and USB interface.

As you may have noticed it takes a **lot** longer to build than the `video` or `net` targets.